src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  let arrCss = sourceString.split(';\n');
+
+  arrCss = arrCss.filter(properties => properties.includes(':'));
+
+  arrCss.forEach(properties => {
+    const [property, value] = properties.split(':');
+
+    result[property.trim()] = value.trim();
+  });
```

Comment:
very good. but you should know that we can also use reduce method here to collect all data in the object